source/leuven/wice_2_rocky9.rst

```diff
@@ -0,0 +1,105 @@
+
+.. note::
+
+   Be aware that toolchains older than 2021a will no longer be available after Genius has been decommissioned.
+
+As always you can contact hpcinfo@kuleuven.be in case you have questions or remarks.
+
+.. _how to test:
+
+How to test
+-----------
+
+In order to give users the chance to test out the new operating system,
+some nodes of the batch, batch_sapphirerapids, gpu_a100 and gpu_h100 partitions
+have alreay been upgraded. To submit jobs to the upgraded nodes,
+simply submit your job with the **--reservation=rocky9_pilot** option to one of the partitions::
+
+   $ sbatch --account=lp_myproject --clusters=wice --reservation=rocky9_pilot\
+            --nodes=2 --ntasks-per-node=72 --time=2:00:00 myjobscript.slurm
+
+Without mentioning a partition you will arrive on the default batch partition which contains IceLake nodes.
+
++-----------------------+-------------------+-----------------+
++ partition             + CPU/GPU type      + number of nodes +
++=======================+===================+=================+
++ batch,batch_icelake   + IceLake           +              12 +
++-----------------------+-------------------+-----------------+
++ batch_sapphirerapids  + Sapphire Rapids   +              12 +
++-----------------------+-------------------+-----------------+
++ gpu_a100              + IceLake / A100    +               1 +
++-----------------------+-------------------+-----------------+
++ gpu_h100              + genoa / H100      +               1 +
++-----------------------+-------------------+-----------------+
+
+This number of nodes is at the start of the test phase. To get an up-to-date overview you can query the system::
+
+   $ scontrol -M wice show reservation rocky9_pilot --json | jq ".reservations[0].node_list"
+
+.. _expected impact:
+
+Expected impact
+---------------
+
+The impact of this upgrade will be small for most users. In case
+you are only using centrally installed modules, the modules that are
+appropriate for the new operating system will be loaded automatically
+when necessary, assuming you are using the cluster modules correctly,
+see :ref:`module system<leuven_module_system>`.
+
+.. note::
+
+   When you are compiling your own software (which can also be the case
+   if you have installed your own Python or R packages), it might be necessary
+   to recompile on a node with the new operating system. If your compiled
+   software does not link to system libraries, it is possible the old
+   executables will still work. In case of doubt, we recommend to recompile.
+   It can be handy to make use of the ${VSC_OS_LOCAL} variable, which will
+   be set to rocky8 or rocky9 depending on the node you are on.
+
+Known issues
+------------
+
+In this section all issues are mentioned that we are aware off. 
+We'll update this section in the course of the test period.
+
```

Comment:
There is a remaining typo in there here, but as I originally suggested these two sentences better just be removed to improve the signal to noise ratio ;)

```suggestion
```